test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultDisplayTemplatesTest.cs

```diff
@@ -369,8 +370,10 @@ public void DisplayFor_DoesNotWrapExceptionThrowsDuringViewRendering()
             helper.ViewData["Property1"] = "ViewData string";
 
             // Act and Assert
-            var ex = Assert.Throws<ArgumentException>(() => helper.DisplayFor(m => m.Property1));
-            Assert.Equal(expectedMessage, ex.Message);
+            var ex = ExceptionAssert.ThrowsArgument(
+                () => helper.DisplayFor(m => m.Property1),
+                null,
+                expectedMessage);
```

Comment:
No need for any change here. Exception is `throw`n a few lines above and has a fixed message.